analytics/Makefile

```diff
@@ -7,6 +7,8 @@ REPO ?= simpler-grants-gov
 SPRINT_PROJECT ?= 13
 ROADMAP_PROJECT ?= 12
 OUTPUT_DIR ?= data
+CONFIG_DIR ?= config
+PROJECT_CONFIG_FILE ?= $(CONFIG_DIR)/github-projects.json
```

Comment:
@coilysiren We don't need to turn this on in AWS step functions yet, but when we're ready to turn on the corresponding step functions, is there anything special we need to do to reference a version controlled config file in the same repo as the source application?

IIRC we had to update the path to the mounted volume for the `OUTPUT_DIR,` but I wasn't sure if we needed to something similar for this config file